app/lib/slack/parses_slash_args_string.rb

```diff
@@ -0,0 +1,25 @@
+require "shellwords"
+
+module Slack
+  class ParsesSlashArgsString
+    SlashArgsParseResult = Struct.new(:subcommand, :args, keyword_init: true)
```

Comment:
**nitpick**: What are your thoughts on 

* `ParsesSlashArgsString` being more closely tied to its objective as `ParsesSlashCommand`
* `SlashArgsParseResult` being something like `ParseResult` since we know what it's referring to within the class. I thought about `ParsedSlashCommand`, but the singular character different makes me a little hesitant.

Reply:
Yeah, I shied away from `ParsesSlashCommand` because this is not, strictly speaking, the full slash command. The slash command (depending on how you look at it) at minimum has the command itself (`/doubleup`) which isn't part of this string. IMO actual "slash command" is broader than that and also has a bunch of other information in the payload. My intent here was to call out that A) this only parses the additional arguments that are passed as part of the full slash command and B) this is parsing a singular thing, not an "args" collection. I could get behind `ParsesSlashCommandArgs` to make it a little more specific though that doesn't really cover point B above.

Reply:
> `SlashArgsParseResult` being something like `ParseResult`
I don't have a super strong opinion, though I generally lean toward more verbose naming. I'm fine with `ParseResult`.

Reply:
That makes sense. I agree `Args` is a good suffix there then for something like `ParsesSlashCommandArgs` or `ParsesCommandArgs` if we want to decouple from the "Slash" nomenclature.